images/build/Dockerfiles/gitHubActions.Dockerfile

```diff
@@ -62,13 +62,16 @@ COPY --from=buildscriptgenerator /opt/buildscriptgen/ /opt/buildscriptgen/
 ARG BUILD_DIR="/opt/tmp/build"
 ARG IMAGES_DIR="/opt/tmp/images"
 RUN ${IMAGES_DIR}/build/installHugo.sh
+RUN "source ${BUILD_DIR}/build/__common.sh" 
```

Comment:
have another script (say foo.sh) which will use the function from common.sh. inside foo.sh you need to source common.sh.

nit: do not add new RUN steps as it increases layers, source it between line 73 and 74